<launch>
  <!--
  	This pushes the local PYTHONPATH into the launch file, so that the NaoQI API is found.
    You need to add the Nao's API dir to your PYTHONPATH so that the modules are found.
  -->
  <env name="PYTHONPATH" value="$(env PYTHONPATH)" />

  <arg name="nao_ip" default="$(optenv NAO_IP 127.0.0.1)" />
  <arg name="nao_port" default="$(optenv NAO_PORT 9559)" />

  <node pkg="leonao" type="inv_kinematic" name="inv_kinematic" required="true"/>
  <node pkg="leonao" type="picture_painter.py" name="picture_painter" required="true" args="$(arg nao_ip) $(arg nao_port)"/>
  <node pkg="leonao" type="picture_taker.py" name="picture_taker" required="true" args="$(arg nao_ip) $(arg nao_port)"/>

  <node pkg="leonao" type="main_controller.py" name="main_controller" required="true" args="$(arg nao_ip) $(arg nao_port)" output="screen"/>

  <include file="$(find nao_apps)/launch/tactile.launch" />
</launch>
